src/createServer.js

```diff
@@ -1,8 +1,33 @@
 'use strict';
 
+const http = require('http');
+const url = require('url');
+const { httpGetMainPage } = require('./routes/httpGetMainPage');
+const { httpVerifyRequest } = require('./routes/httpVerifyRequest');
+const { httpCompress } = require('./routes/httpComress');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer(async (req, res) => {
+    const method = req.method;
+    const normalizedText = new url.URL(req.url, `http://${req.headers.host}`)
+      .pathname;
+
+    if (normalizedText === '/') {
+      await httpGetMainPage(res);
+
+      return;
+    }
+
+    const verifiedPath = httpVerifyRequest(res, normalizedText, method);
+
+    if (!verifiedPath) {
+      return;
+    }
+
+    if (method === 'POST' && normalizedText === '/compress') {
```

Comment:
'/' and '/compress' move in constants.
Also, all methods ( 'POST' and others) move to constants. 
Finally, move all statuses in constants. ( for all files) 
Then use their in all files. 
Thanks)

Reply:
@DanilWeda I will make it on my laptop, because I like this approach, but I will not commit new changes)
Is it ok for you?

Tests work awful I am afraid to move something while the result is green)))

Reply:
After the first attempt, I had 3 errors.
Then I did nothing and only one mistake is left)
But as you can see from the screen - everything works properly)

![image](https://github.com/mate-academy/node_compression-app/assets/102412814/47f07dc2-0ea7-44af-ba12-04a83142aa41)